custom_updater.json

```diff
@@ -1,16 +1,16 @@
 {
     "circadian_lighting": {
-        "updated_at": "2020-06-11",
-        "version": "1.0.13",
+        "updated_at": "2021-04-29",
+        "version": "2.1",
         "local_location": "/custom_components/circadian_lighting/__init__.py",
-        "remote_location": "https://raw.githubusercontent.com/claytonjn/hass-circadian_lighting/master/custom_components/circadian_lighting/__init__.py",
-        "visit_repo": "https://github.com/claytonjn/hass-circadian_lighting",
-        "changelog": "https://github.com/claytonjn/hass-circadian_lighting/releases",
+        "remote_location": "https://raw.githubusercontent.com/robertomano24/circadian_lighting/master/custom_components/circadian_lighting/__init__.py",
```

Comment:
I see you created a fork of this repo, maybe let's keep the original one pointing to the same repo [here](https://raw.githubusercontent.com/claytonjn/hass-circadian_lighting)?

Also, you can move the repo name to a variable to have the ability to change it later on.

Reply:
Agreed. I will change back. That happened during an exploration of launching a separate tool, assuming this particular tool is no longer being maintained. Until then, it will go back! Thanks

Reply:
That should do it. Good catch. Thanks!

Reply:
I hope the author comes back to review this, but I'll use your repo for now. Thanks for creating and maintaining the fork